android-sdk/src/main/java/com/checkout/android_sdk/CheckoutAPIClient.java

```diff
@@ -168,6 +192,11 @@ public CheckoutAPIClient setGooglePayListener(OnGooglePayTokenGenerated listener
     public String getCorrelationID() {
         return mCorrelationID;
     }
+
+    public Environment getEnvironment() {
+        return mEnvironment;
+    }
```

Comment:
can this be private?

Reply:
mEnvironment is private variable in CheckoutApiClient. To support for testing, I have added public methods to expose this variable.

Reply:
I don't think this method is used in any of the tests. Can we remove it?